cellprofiler/modules/measuretexture.py

```diff
@@ -84,6 +87,39 @@
       relationship between specific intensity values. It is a complementary
       value to InfoMeas1 and is on a different scale.
 
+**Note**: each of the above measurements are computed for different 
+'directions' in the image, specified by a series of correspondence vectors. 
+These are indicated in the results table in the *scale* column as n_00, n_01,
+n_02... for each scale *n*. In 2D, the directions and correspondence vectors *(y, x)* 
+for each measurement are given below:
+
+- _00 = horizontal, 0 degrees   (0, 1)
+- _01 = diagonal, 135 degrees   (1, 1)
```

Comment:
Can we add the diagonal directions here from mahotas? I think that adds an extra layer of intuitability